_drafts/oryx/browser-compatibility.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "Oryx: Supported browsers, OSes, and resolutions"
+description: Browsers, OSes, and resolutions supported by Oryx
+template: concept-topic-template
+---
+
+Oryx supports and is being tested on all modern browsers, operating systems and resolutions.
+
+## Browser compatibility
+
+| BROWSER | VERSION | DESKTOP SUPPORT | MOBILE SUPPORT |
+| --- | --- | --- | --- |
+| Safari | 2 most recent major versions | &#9989; | &#9989; |
+| Chrome | 2 most recent major versions | &#9989; | &#9989; |
+| Edge | 2 most recent major versions | &#9989; | |
+| Firefox | 2 most recent major versions | &#9989; | |
```

Comment:
why are Edge and FF not supported on mobile? 

What about adding a section or link to the screen size support?

Reply:
> why are Edge and FF not supported on mobile?

We are not testing our components in Edge and FF on mobile (actually, we are not checking them on Edge and FF on desktops either) - that's why they are not supported. We do not have a proof that components/sites work well in these browsers, and just having them listed in the `.browserlistrc` file does not prove that it works(
---

> What about adding a section or link to the screen size support?

You've asked to remove it from here :) https://github.com/spryker/spryker-docs/pull/1728#discussion_r1163685252
>We could skip talking about screen resolutions here as well. It will be covered else where anyway (inside styles/design tokens).

Reply:
Then should we change references to "support"? We do intent stuff to work on these browsers, and we expect things to work. It might cause bugs, that's actually great if they are reported because of this docs. We cannot end up with a docs that does not guarantee usage on these browser, customers will not use our stuff.